### DISCUSSION TITLE
TW-Elements Components don't working with Transloco

### DISCUSSION THREAD

**Discussion by matti011g** [mdbootstrap/TW-Elements#2302] [Support from community] [answered]
Everything inside a tag with this transloco code <div ***transloco="let t;"** class="px-14">  doenst work properly, I want to use a date-picker component and it breaks, any help?

**Comment by iprzybysz** [comment]
Hi @matti011g, please describe your issue clearly, and add details about it - provide a snippet of your code, configuration and content of your error messages if there are any.

**Comment by matti011g** [comment]
Hi @iprzybysz, i'm working with angular 16, and using "@ngneat/transloco": "^6.0.0" for translations, and I don't have any error message but every component I use inside this tag <div *transloco="let t;" ></div>  breaks, I already did the test by moving the component outside the tag and it works

**Reply by iprzybysz** [reply]
Unfortunately, we aren't able to help you with Transloco, because we don't know it. Maybe someone from the community had a similar issue and will help you with it. We also have an [Internationalization guide](https://tw-elements.com/docs/standard/getting-started/internationalization/) - which may help you.

**Comment by matti011g** [comment, marked as answer]
I found the solution, its declare trnasloco inside the tag directly like this :  <h3>{{ "checkout.choose-date.title-1" | transloco }}</h3>
